src/helpers/get-week-number.ts

```diff
@@ -1,9 +1,15 @@
-export const getWeekNumber = (d: Date) => {
+export const getYearWeek = (d: Date) => {
```

Comment:
Maybe we can add a return type here, I think it's easy to accidentally return some number instead.

```suggestion
export const getYearWeek = (d: Date): [string, string] => {
```